README.md

```diff
@@ -36,7 +36,8 @@ Options:
         --burst-max                                          Max level for bursting metrics (int) [default: 80]
         --prometheus=<true|false>                            Enable prometheus metrics [default: false]
         --zone                                               Zone for log [default: dev]
-        --region                                             Region for log [default: dev]   
+        --region                                             Region for log [default: dev]
+        --secrets-volume                                     Secrets Volume Path [default: secrets]   
```

Comment:
Mind updating the documentation to show that it's from the root directory like `/secrets`? That would help clear some confusion on where the path is actually defined